target/linux/ath79/image/generic.mk

```diff
@@ -154,7 +161,8 @@ TARGET_DEVICES += buffalo_wzr-hp-ag300h
 
 define Device/buffalo_wzr-hp-g302h-a1a0
   ATH_SOC := ar7242
-  DEVICE_TITLE := Buffalo WZR-HP-G302H A1A0
+  DEVICE_VENDOR := Buffalo
+  DEVICE_MODEL := WZR-HP-G302H A1A0
```

Comment:
A1A0 might correspond to the revision (=DEVICE_VARIANT) here (rev. A1 and rev. A0?)